core/server/models/webhook.js

```diff
@@ -7,6 +7,16 @@ let Webhook,
 Webhook = ghostBookshelf.Model.extend({
     tableName: 'webhooks',
 
+    defaults() {
+        return {
+            api_version: 'v2'
```

Comment:
Does the v0.1 api need to explicitly set this to `null` now?

Reply:
It will not need to, we'll only be using `api_version` later for serialization purposes as in which payload to send, and all new user-facing webhooks created will always default to sending this. Anything which v0.1 API creates/sends, like Zapier, will not have anything to do with `api_version` so does not need to explicit set it

Reply:
Sorry I see what you mean now, right so this will only work under the assumption that payload for subscribers does not change in v2, as is the case right now, but might not be ahead. 🤔 Not sure if changing v0.1 api is ideal, but yeah def needs more thought. @kevinansfield Wdyt about this?

Reply:
Yeah that's all i'm thinking :/ I think changing v0.1 is fine - we should just need an `Object.assign({api_version: 'v0.1'}, blah...)` here: https://github.com/TryGhost/Ghost/blob/master/core/server/api/v0.1/webhooks.js#L84

Reply:
I'd leave it as-is because v2 will be the default for other webhook events in the future.

Zapier is the only webhook consumer at the moment and it's under our control, in the unlikely event that the subscriber payload changes drastically in v2 then we can update Zapier and push it out to everyone using it ready for the change.

Reply:
> it's under our control
v. good point :+1:

Reply:
That def solves for any new webhooks created via Zapier integration, but for any existing webhooks in db, migration script will auto create and set `api_version` field to `v2` as its default. So it leads to the same problem that in future it will send v2 payload for older webhooks. 

So then maybe its cleaner to set `api_version` default as v0.1 and update the v2 webhooks controller to always set it to v2 (assuming ofc Zapier still keeps using v0.1 api). That ensures migration doesn't break anything existing, and all new webhooks created via v2 APIs have v2 payload which is expected

Reply:
Sorry if I wasn't clear 😬 We won't have any webhooks with v0.1 payloads now or in the future, _all_ webhooks are v2 by default, even the existing ones.

Reply:
@kevinansfield oops sorry, it was clear, I actually missed your earlier comment as hadn't refreshed, so my comment was in context of earlier discussion about v0.1 change. 🙈 Above makes sense 👍